Stopwatch/Stopwatch.swift

```diff
@@ -24,11 +25,27 @@ class Stopwatch {
     }
     
     func start() {
-        startTime = Date()
+        if !isRunning {
+            startTime = Date()
+        }
     }
     
     func stop() {
+//        This is to calculate the original starting time
+//        resumeTime = Date().addingTimeInterval(-elapsedTime)
+        resumeTime = elapsedTime
```

Comment:
You should probably check if the stopwatch is running here before executing the next two lines.